src/lib389/cli/openldap_to_ds

```diff
@@ -0,0 +1,188 @@
+
+    slapcat -f /etc/openldap/slapd.conf -b SUFFIX -l /root/suffix.ldif
+
+You must already have a 389 Directory Server you want to import into. You can create
+this with the `dscreate` tool. Data and configuration in this instance WILL be
+modified or removed (ie potentially destructive actions).
+
+It is strongly advised you test this tool on a non-production system first to be
+sure the process and changes are understood.
+
+This only needs to be run on the first-instance in a 389 Directory Server topology. All
+other replicas should be configured from this instance post migration.
+""")
+parser.add_argument('-v', '--verbose',
+                    help="Display verbose operation tracing during command execution",
+                    action='store_true', default=False, dest='verbose')
+
+parser.add_argument('instance',
+        help="The name of the ds instance to have openldap data migrated into",
+    )
+parser.add_argument('slapd_config',
+        help="The path to the slapd.d configuration that will be migrated",
+    )
+# Migration options
+parser.add_argument('--confirm',
+        default=False, action='store_true',
+        help="Confirm that you want to apply these actions to the ds instance. By default no actions are taken."
+)
+parser.add_argument('--ignore-overlay', nargs='*',
+        help="Ignore the following overlays from having their configuration migrated, if possible."
+)
+parser.add_argument('--ignore-schema-oid', nargs='*',
+        help="Ignore the following schema attribute or class OIDS from being migrated. This *may* create inconsistent schema which could cause the migration to fail. Use with caution."
+)
+parser.add_argument('--ignore-attribute', nargs='*',
+        help="Ignore the following attributes from entries that are loaded from the ldif. For example, you may not want to import userPassword hashes."
+)
+parser.add_argument('--no-overlays',
+        default=True, action='store_false',
+        help="Do not migrate any overlays."
+)
+parser.add_argument('--no-schema',
+        default=True, action='store_false',
+        help="Do not migrate the schema. This may cause the ldif import to fail."
+)
+parser.add_argument('--no-indexes',
+        default=True, action='store_false',
+        help="Do not create any indexes as defined in slapd.d"
+)
+
+
+# General options
+parser.add_argument('-D', '--binddn',
+        help="The account to bind as for executing operations",
+        default=None
+    )
+parser.add_argument('-w', '--bindpw',
+        help="Password for binddn",
+        default=None
+    )
+parser.add_argument('-W', '--prompt',
+        action='store_true', default=False,
+        help="Prompt for password for the bind DN"
+    )
+parser.add_argument('-y', '--pwdfile',
+        help="Specifies a file containing the password for the binddn",
+        default=None
+    )
+parser.add_argument('-Z', '--starttls',
+        help="Connect with StartTLS",
+        default=False, action='store_true'
+    )
```

Comment:
Probably, worth specifying in the description for which server the credentials are: OpenLDAP or 389-ds-base (I understand that we process an LDIF file for OpenLDAP, mostly, but it's better to be explicit than implicit in such docs, IMHO)